manim/renderer/cairo_renderer.py

```diff
@@ -84,24 +84,23 @@ def play(self, scene, *args, **kwargs):
 
         scene.compile_animation_data(*args, **kwargs)
 
-        # If skip_animations is already True, we can skip all the caching process.
-        if not config["disable_caching"] and not self.skip_animations:
-            hash_current_animation = get_hash_from_play_call(
-                scene, self.camera, scene.animations, scene.mobjects
-            )
-            if self.file_writer.is_already_cached(hash_current_animation):
-                logger.info(
-                    f"Animation {self.num_plays} : Using cached data (hash : %(hash_current_animation)s)",
-                    {"hash_current_animation": hash_current_animation},
-                )
-                self.skip_animations = True
-        else:
-            hash_current_animation = f"uncached_{self.num_plays:05}"
-
         if self.skip_animations:
             logger.debug(f"Skipping animation {self.num_plays}")
             hash_current_animation = None
-
+        else:
+            if config["disable_caching"]:
+                logger.debug("Caching disabled.")
```

Comment:
Thought's on making it info?
```suggestion
                logger.info("Caching disabled.")
```

Reply:
Why not, thanks